### ISSUE TITLE
Select ARIA violation around not having an accessible name

### ISSUE THREAD

**Issue by hergaiety** [miguelcobain/ember-paper#1171]
The [ember-paper Select](https://miguelcobain.github.io/ember-paper/#/components/select) component renders a wrapper element `<md-input-container ...attributes>` that absorbs any attributes, not allowing any `aria-label` or `aria-describedby` or similar accessibility attributes to be passed to the underlying `<md-select>` that Ember Power Select outputs. This results in a `[serious]: ARIA input fields must have an accessible name` https://dequeuniversity.com/rules/axe/3.5/aria-input-field-name?application=axeAPI.

This is even true if you supply a `@label="my label"`.

Running a `a11yAudit` with popular [`ember-a11y-testing` addon](https://github.com/ember-a11y/ember-a11y-testing) fails with the above rule violation. (For a full reproduction, I awaited for a click on the dropdown to show the options first)

![Screenshot_2020-10-20_15-45-22](https://user-images.githubusercontent.com/1664093/96642109-4d64d380-12eb-11eb-990a-95e2ecb06ee5.png)
